strax/context.py

```diff
@@ -904,6 +904,41 @@ def get_array(self, run_id: ty.Union[str, tuple, list],
             results = [x.data for x in source]
         return np.concatenate(results)
 
+    def get_accum_array(self,
+                        run_id: str,
+                        targets,
+                        fields,
+                        **kwargs):
+        """
+        Computes targets for run_id and accumulated result for specified
+        fields.
+
+        :param fields: Column names of the data which should be
+            accumulated.
+        """
+        ind = 0
```

Comment:
Unless I misread something, it doesn't look like this ever gets incremented, so the result will always be reset. You could wrap get_iter in enumerate if you want to know the chunk index.

Reply:
Oh yes, good catch thanks.